src/views/partials/gallery.ejs

```diff
@@ -14,9 +14,9 @@
         </li>
     </ul>
     <div class="gallery-wrapper-miniture">
-        <img id="0" class="gallery-img active" src=<%= product.product_gallery.image1 %> alt="product image 1" />
-        <img id="1" class="gallery-img" src=<%= product.product_gallery.image2 %> alt="product image 3" />
-        <img id="2" class="gallery-img" src=<%= product.product_gallery.image3 %> alt="product image 2" />
-        <img id="3" class="gallery-img" src=<%= product.product_gallery.image4 %> alt="product image 4" />
+        <img id="0" tabindex="0" class="gallery-img active" src=<%= product.product_gallery.image1 %> alt="product image 1" />
+        <img id="1" tabindex="0" class="gallery-img" src=<%= product.product_gallery.image2 %> alt="product image 3" />
+        <img id="2" tabindex="0" class="gallery-img" src=<%= product.product_gallery.image3 %> alt="product image 2" />
+        <img id="3" tabindex="0" class="gallery-img" src=<%= product.product_gallery.image4 %> alt="product image 4" />
```

Comment:
I do not think it is bad for accessibility, since it is `tabindex = 0`, and this follows the relative flow, now I also need it since without this the elements of the gallery would not be focused

Reply:
I see your point it is ok. In other hand I see this code can be generated with a loop structure so no markup is duplicated, check the id attribute as well it can be easy be duplicated and id="1" is too general, please change to "gallery-img-1".

Reply:
I'm not quite sure how could I duplicate this ` src=<%= product.product_gallery.image2 %>`

Reply:
you can do `<% for (const index = 1; index<= 4; index++) { %>` and then `<img id=<%= index %> tabindex="0" class="gallery-img" src=<%= product.product_gallery['image' + index] %> alt="product image "<%= index %> />`

Reply:
Done